src/web/services/user/profile.rs

```diff
@@ -87,7 +88,7 @@ async fn profile(
                 // Return early if there was an error.
                 // Otherwise we can go forward and check for the user's membership in the RCOS
                 // Discord server.
-                return template.in_page(&req, page_title).await;
+                return template.in_page(&req, page_title.clone()).await;
```

Comment:
Why clone here? Ownership will not get moved after returning. Does the compiler error otherwise?

Reply:
I was having some issues with ownership and put that while debugging. After thinking about it, it seems unnecessary since it returns so I'll remove.